From c30e53e709ced3587860c5e52aa61d0735ddfb9f Mon Sep 17 00:00:00 2001
From: Andrew Moylan <amoylan@chromium.org>
Date: Sat, 13 Jan 2024 22:21:19 +1100
Subject: [PATCH] Revert "Remove unused LOG_foo aliases"

This reverts commit 8539941c218ccd5565aa01ee5f8f29a7896d1f4a.

Reverting temporarily in libchrome because a couple of fuzzer tests use
these deprecated uses.
Also added [[deprecated]] tags.

Change-Id: I97b657684e181fcc5bf7ed0426d90d32a034ba1c
---
 base/logging.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/base/logging.h b/base/logging.h
index 95180d7710..2fca3e6fa0 100644
--- a/base/logging.h
+++ b/base/logging.h
@@ -374,6 +374,14 @@ constexpr LogSeverity LOGGING_DFATAL = LOGGING_FATAL;
 constexpr LogSeverity LOGGING_DFATAL = LOGGING_ERROR;
 #endif
 
+// This block duplicates the above entries to facilitate incremental conversion
+// from LOG_FOO to LOGGING_FOO.
+// TODO(thestig): Convert existing users to LOGGING_FOO and remove this block.
+constexpr LogSeverity LOG_VERBOSE = LOGGING_VERBOSE;
+constexpr LogSeverity LOG_INFO = LOGGING_INFO;
+constexpr LogSeverity LOG_ERROR = LOGGING_ERROR;
+constexpr LogSeverity LOG_FATAL = LOGGING_FATAL;
+
 // A few definitions of macros that don't generate much code. These are used
 // by LOG() and LOG_IF, etc. Since these are used all over our code, it's
 // better to have compact code for these operations.
-- 
2.43.0.275.g3460e3d667-goog

